egs/wsj/s5/utils/combine_ali_dirs.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Copyright 2016  Xiaohui Zhang  Apache 2.0.
+
+# This script operates on alignment directories, such as exp/tri4a_ali
+
+# Begin configuration section. 
+cmd=run.pl
+extra_files= # specify addtional files in 'src-data-dir' to merge, ex. "file1 file2 ..."
+ref_data_dir= # data directory to be used as reference for rearranging alignments
+num_jobs= # number of jobs used to split the reference data directory.
```

Comment:
I think the common practice is to have default values for optional arguments like `num_jobs=4`.